.github/workflows/docker-cloudstack-simulator.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Docker Image Build
+
+on:
+  push:
+    branches:
+      - main
+    tags:        
+      - 4.*
+      - 5.*
+
+permissions:
+  contents: read
+
+concurrency:
```

Comment:
@weizhouapache could you add a code or check so this only runs on apache/cloudstack and not on any other forks. cc @DaanHoogland 

In fact there are jobs such as sonar and codecov too which shouldn't run on forks.

Reply:
@rohityadavcloud 
thanks. good idea. updated this PR